main.py

```diff
@@ -1,86 +1,110 @@
-import urllib3
-import json
 import gzip
+import json
 import sqlite3
-from tabulate import tabulate
 from datetime import date, datetime
+
+import urllib3
+from tabulate import tabulate
+
 http = urllib3.PoolManager()
 import requests
 
 
-
 def pretty_print_data(data):
-    '''Print data retrieved from database in tabular format'''
-    print(tabulate(data, headers=["Domain Code","Page Title", "View Count","Response size (bytes)"]))
+    """Print data retrieved from database in tabular format"""
+    print(
+        tabulate(
+            data,
+            headers=[
+                "Domain Code",
+                "Page Title",
+                "View Count",
+                "Response size (bytes)",
+            ],
+        )
+    )
+
 
 def format_number(number):
-    '''Return string representation of a number also if no less than 9 add leading zeros'''
-    if number < 9 :
-        return str("0"+ str(number))
+    """Return string representation of a number also if no less than 9 add leading zeros"""
+    if number < 9:
+        return str("0" + str(number))
     else:
         return str(number)
+
+
 def create_Url():
-    '''Ask user for date and time of which user wants to see page view data and return file downloaded data'''
-    year = format_number(int(input('Enter a year: ')))
-    month = format_number(int(input('Enter a month: ')))
-    day = format_number(int(input('Enter a day: ')))
-    hour = format_number(int(input('Enter the hour: ')))
-    newUrl = "https://dumps.wikimedia.org/other/pageviews/{year}/{year}-{month}/pageviews-{year}{month}{day}-{hour}0000.gz".format(year=year, month=month, day=day, hour=hour)
-    return makeRequest(newUrl,str(year+month+day))
-def makeRequest(url,filename):
-    '''Download file and rename and save it in local directory''' 
+    """Ask user for date and time of which user wants to see page view data and return file downloaded data"""
+    year = format_number(int(input("Enter a year: ")))
+    month = format_number(int(input("Enter a month: ")))
+    day = format_number(int(input("Enter a day: ")))
+    hour = format_number(int(input("Enter the hour: ")))
+    newUrl = "https://dumps.wikimedia.org/other/pageviews/{year}/{year}-{month}/pageviews-{year}{month}{day}-{hour}0000.gz".format(
+        year=year, month=month, day=day, hour=hour
+    )
+    return makeRequest(newUrl, str(year + month + day))
+
+
+def makeRequest(url, filename):
+    """Download file and rename and save it in local directory"""
```

Comment:
This doc string is not wrong. But it is also not fit for production documentation process. Use following style instead
```python
"""Download file and rename and save it in local directory

    Args:
        url (str): _description_
        filename (str): _description_

    Returns:
        _type_: _description_
    """    
```
This template can be auto generated in VSCode